.rubocop_disabled.yml

```diff
@@ -35,30 +64,6 @@ Style/Encoding:
   StyleGuide: '#utf-8'
   Enabled: false
 
-Layout/FirstArrayElementLineBreak:
-  Description: >-
-                 Checks for a line break before the first element in a
-                 multi-line array.
-  Enabled: false
-
-Layout/FirstHashElementLineBreak:
-  Description: >-
-                 Checks for a line break before the first element in a
-                 multi-line hash.
-  Enabled: false
-
-Layout/FirstMethodArgumentLineBreak:
-  Description: >-
-                 Checks for a line break before the first argument in a
-                 multi-line method call.
-  Enabled: false
-
-Layout/FirstMethodParameterLineBreak:
-  Description: >-
-                 Checks for a line break before the first parameter in a
-                 multi-line method parameter definition.
-  Enabled: false
-
```

Comment:
I just downloaded the new version of the default config from the rubocop repo at the `0.49.1` tag.